day1/day_1.erl

```diff
@@ -0,0 +1,36 @@
+-module(day_01).
+-export([calc_fuel_for_modules/0, calc_total_fuel/0]).
+
+read_input(FileName) ->
+    {ok, Data} = file:read_file(FileName),
+    lists:delete(<<>>, binary:split(Data, [<<"\n">>], [global])).
+
+process_input(L) ->
+    [ list_to_integer(binary_to_list(X)) || X <- L ].
```

Comment:
If you're using a newer Erlang version, you should have access to binary_to_integer/integer_to_binary pairs directly.

[erldocs](https://erldocs.com/21.0/) is a good resource for searching functions/modules.

Reply:
I updated my solution to use `file:read_line` rather than `file:read_file` which was the source of most of my problems. This change allowed me to remove newlines, and convert the binaries to integers while only having to traverse the input one time.